lib/devices/device.js

```diff
@@ -41,6 +43,13 @@ class Device {
   set(options) {
     assert(options);
 
+    if (options.logger != null)
```

Comment:
Does anything type coerce to `null`? I don't think so, but using `!==` and `===` are safer

Reply:
`null` and `undefined`.